genesyscloud/resource_genesyscloud_telephony_providers_edges_site.go

```diff
@@ -494,6 +584,22 @@ func deleteSite(ctx context.Context, d *schema.ResourceData, meta interface{}) d
 	})
 }
 
+func buildMediaRegionList(d *schema.ResourceData) *[]string {
```

Comment:
The function `buildSdkStringListFromInterfaceArray` does this job I'm fairly sure.